scripts/all_repos.py

```diff
@@ -59,7 +59,7 @@
     org_vice_repos[org] = repos
 
 # Write the list of dictionaries to a JSON file
-with open("artifacts.json", "w") as f:
+with open("../artifacts.json", "w") as f:
```

Comment:
```suggestion
with open("artifacts.json", "w") as f:
```

We will run the script from root of this repo, not from inside scripts folder.